source/administration/mobile-changelog.md

```diff
@@ -1,5 +1,44 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.43.0 Release
+- Release Date: May 16, 2021
+- Server Versions Supported: Server v5.31.3+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.31.3 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.25 has ended and upgrading to server ESR v5.31.3 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### Support for Apps Framework (Developer Preview)
+ - Apps Framework is now supported on the mobile client and allows developers to create Apps which have interactivity and buttons on mobile. You can learn more at https://developers.mattermost.com/integrate/apps/.
+
+### Improvements
+ - Added avatars to the sidebar Direct Messages and channel info.
+ - On Websocket reconnect, instead of getting all the groups, only the updated groups are now fetched.
+ - Team channels including archived channels are now fetched since the last known timestamp.
+
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue where appended skin tone emoji didn't render on the app.
+ - Fixed an issue where users were unable to get to the last line when editing a very long post.
+ - Fixed the user autocomplete by adding the current user to the list.
+
+#### Android specific
+ - Fixed an issue where archived teams were accessible and functional.
```

Comment:
Is this accurate? What are archived teams? Is it supposed to say archived channels?

Reply:
Hi Ben, yes it is **teans**. You can archive a team using the server cli :)